meta-ros-common/conf/ros-distro/ros-distro.conf

```diff
@@ -14,7 +14,7 @@ ROS_DISTRO_COMPAT = "melodic dashing eloquent"
 
 # This variable is updated with the new release series as the first commit to [master] after the branch for the current release
 # series is created.
-ROS_OE_RELEASE_SERIES = "langdale"
+ROS_OE_RELEASE_SERIES = "kirkstone"
```

Comment:
I don't believe we want to move to kirkstone in master. It'll probably better to move things first to `kirkstone` branch…

Reply:
You are correct since langdale is more recent. 
@vmayoral Can you please create a kirkstone branch for me and so I can push this PR into there?